<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1057</width>
    <height>794</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="main_layout">
      <item>
       <layout class="QVBoxLayout" name="top_layout">
        <item>
         <layout class="QHBoxLayout" name="dir_layout">
          <item>
           <widget class="QToolButton" name="toolButton">
            <property name="text">
             <string>...</string>
            </property>
            <property name="iconSize">
             <size>
              <width>12</width>
              <height>12</height>
             </size>
            </property>
            <property name="arrowType">
             <enum>Qt::LeftArrow</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="path_lineEdit"/>
          </item>
          <item>
           <widget class="QCheckBox" name="subfolders_checkBox">
            <property name="text">
             <string> Include subfolders</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="browse_button">
            <property name="text">
             <string>Browse...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="files_layout">
          <item>
           <widget class="QTreeView" name="folders_treeView"/>
          </item>
          <item>
           <widget class="QTreeView" name="files_treeView"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="bottom_layout">
        <item>
         <layout class="QHBoxLayout" name="actions_layout">
          <item>
           <layout class="QVBoxLayout" name="extensions_layout">
            <item>
             <widget class="QGroupBox" name="extensions_groupBox">
              <property name="title">
               <string>Extensiosn</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <widget class="QCheckBox" name="checkBox_2">
                 <property name="text">
                  <string>CheckBox</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="extensionsButtons_layout">
              <item>
               <widget class="QPushButton" name="allExtension_button">
                <property name="text">
                 <string>All</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="noneExtension_button">
                <property name="text">
                 <string>None</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="ignored_layout">
            <item>
             <widget class="QGroupBox" name="ignored_groupBox">
              <property name="title">
               <string>Ignored</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <widget class="QCheckBox" name="checkBox_3">
                 <property name="text">
                  <string>CheckBox</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="ignoredButtons_layout">
              <item>
               <widget class="QPushButton" name="allIgnored_button">
                <property name="text">
                 <string>All</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="noneIgnored_button">
                <property name="text">
                 <string>None</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QGroupBox" name="actions_groupBox">
            <property name="title">
             <string>Actions</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QRadioButton" name="radioButton">
               <property name="text">
                <string>Delete</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButton_3">
               <property name="text">
                <string>Copy</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButton_2">
               <property name="text">
                <string>Move</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="info_layout">
            <item>
             <widget class="QLabel" name="totalFiles_label">
              <property name="text">
               <string>Total Files : 1</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="selectedFiles_label">
              <property name="text">
               <string>Selected Files : 0 ~ 0.0 MB</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QVBoxLayout" name="process_layout">
            <item>
             <widget class="QCheckBox" name="refresh_checkBox">
              <property name="text">
               <string>Auto refresh</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="refresh_button">
              <property name="text">
               <string>Refresh</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="process_button">
              <property name="text">
               <string>Process</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QProgressBar" name="progressBar">
          <property name="value">
           <number>24</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1057</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionSet_Default_Style"/>
    <addaction name="actionSet_Dark_Style"/>
   </widget>
   <addaction name="menuView"/>
  </widget>
  <action name="actionSet_Default_Style">
   <property name="text">
    <string>Set Default Style</string>
   </property>
  </action>
  <action name="actionSet_Dark_Style">
   <property name="text">
    <string>Set Dark Style</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
